src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  margin-right: 50px;
```

Comment:
if you open Figma design you will see that margin is 50px - but the border is included in this value (2 times), so your real margin is 48px. look on design once more

Reply:
let put your margin in class -  main__card  you create it to position your card in parent container